index.bs

```diff
@@ -900,7 +900,7 @@ The <dfn>context type</dfn> is the type of the execution context that manages th
 </dl>
 
 <div class="note">
-When the {{MLContext/[[contextType]]}} is set to [=context type/default=] with the {{MLContextOptions}}.{{MLContextOptions/deviceType}} set to {{MLDeviceType/"gpu"}}, the user agent is responsible for creating an internal GPU device that operates within the context and is capable of ML workload submission on behalf of the calling application. In this setting however, only {{ArrayBufferView}} inputs and outputs are allowed in and out of the graph execution since the application has no way to know what type of internal GPU device is being created on their behalf. In this case, the user agent is responsible for automatic uploads and downloads of the inputs and outputs to and from the GPU memory using this said internal device.
+When the {{MLContext/[[contextType]]}} is set to [=context type/default=] with the {{MLContextOptions}}.{{MLContextOptions/deviceType}} set to {{MLDeviceType/"gpu"}}, the user agent is responsible for creating an internal GPU device that operates within the context and is capable of ML workload submission on behalf of the calling application.
```

Comment:
FYI I'm not touching the rest of this paragraph since it relates to #749, which is a can of worms I don't want this PR to open!

Reply:
Should we remove it together with compute() method? Compute() only takes `ArrayBufferView`s.

Reply:
We certainly could. I just figured it would be nice to chunk off bits like this which haven't been true for a while 🤷 Happy to abandon this PR if you'd prefer to remove this alongside `compute()`